cmd/copyEnumeratorInit.go

```diff
@@ -674,6 +674,12 @@ func NewFolderPropertyOption(fromTo common.FromTo, recursive, stripTopDir bool,
 		}
 	}
 
+	// In case of a Blob to File Copy, we will go for all folder copy
+	if fromTo == common.EFromTo.BlobFile() {
```

Comment:
I thought we decided to use the mitigation (customer sets include directory flag) and get to this issue in a later release after we have a solid plan?

@seanmcc-msft  @vibhansa-msft thoughts?

Reply:
We can consider this fix for the next release (not for 10.23). Once we have more clarity on the flow, may be we can improvise the fix.

Reply:
We also need to consider the part where putting such check may take away the capabilities to skip empty folders while copying.